commands/__init__.py

```diff
@@ -170,12 +171,10 @@ class Columns(Enum):
     help="Deployment description",
 )
 @click.option(
-    "-c",
-    "--require-context",
-    multiple=True,
-    default=["+"],
-    help="Context required to be in success state for this deployment to run; "
-    "use a single '-' to require no contexts, or a single '+' to require all",
+    "-f",
+    "--force-deploy",
```

Comment:
Since the command is named `deploy`, I would prefer `--force` as name for the parameter over `--force-deploy` to avoid redundancy.